dashboard/storage/local.py

```diff
@@ -29,3 +31,19 @@ def save_file(self, name: str, content: bytes) -> None:
             raise ValueError("data_folder is not set")
         with open(self.data_folder / name, "wb") as f:
             f.write(content)
+
+    def delete_file(self, name: str, seconds: int = 0) -> None:
+        time.sleep(seconds)
```

Comment:
If you want to maintain the sleeping functionality I'd move it outside the `FileStorage` class so that if someone really wants to delete file with a sleep before hand he has to wrap it with a separate function (as putting it directly in the FileStorage class is quite unintuitive imo)